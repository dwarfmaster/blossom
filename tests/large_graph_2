18 31
3 2
2 5
5 6
6 3
6 2
6 13
5 13
5 12
13 12
13 9
9 12
12 16
16 17
17 13
16 13
12 17
12 11
11 15
15 16
11 14
14 15
11 10
10 7
7 8
8 10
9 8
8 4
4 7
7 0
0 1
1 4
