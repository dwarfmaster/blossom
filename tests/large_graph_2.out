3 -- 2
5 -- 6
13 -- 9
16 -- 17
12 -- 11
14 -- 15
10 -- 7
8 -- 4
0 -- 1
